okapi-common/pom.xml

```diff
@@ -15,7 +15,6 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-core</artifactId>
-      <scope>test</scope>
```

Comment:
Why isn't this `<optional>true</optional>` to allow users of okapi-common to use some other logging library if they want?